lang/cs/Org.Apache.REEF.IO/FileSystem/IFileSystem.cs

```diff
@@ -104,5 +104,11 @@ public interface IFileSystem
         /// <returns></returns>
         /// <exception cref="IOException"></exception>
         IEnumerable<Uri> GetChildren(Uri directoryUri);
+
+        /// <summary>
+        /// Returns Url prefix 
+        /// </summary>
+        /// <returns></returns>
+        string UriPrefix { get; }
```

Comment:
there is a problem here. What if the user already gives prefixed path. We need to automatically detect it inside as markus said.